lab/components/AsyncInlineSvg.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+	<!--
+		hack I temporarily used to fix an issue
+		that seems to have magically fixed itself
+		but I'm keeping this code here in case
+		I wanna experiment more with it later
+	-->
+	<html-fragment
+		v-if="fragment"
```

Comment:
Curious why give the option to use fragment instead of just using out of the box for consistency? What use case is there to not use this as a fragment? 

Seems like it's the more favored option, but you have to opt into it.

Reply:
Yup, you're exactly right, fragment is better and should be the default, the issue is that I was having some problems with getting it to work in my Lab demo which is why I have a toggle to turn it off and on. Ideally, I'd like to ship the fragment-version of AsyncInlineSvg to Website BUT if it's not do-able I think we can live with just having a `span` wrapper around the SVGs. It depends on if @privatenumber has the time to help me debug the issue with `vue-frag` I mentioned in this comment: https://github.com/square/maker/pull/528#discussion_r1116023900

Reply:
Note: this code is part of a Lab demo and is not shipped as part of Maker, it's just a toy thing I was experimenting with. I intend to implement a cleaned up version of AsyncInlineSvg in Website, i don't think the component itself needs to exist as part of Maker.

Reply:
👍 Just wanted to throw some quick notes in while I was reviewing, not a blocker at all